### DISCUSSION TITLE
I can't click on "auto sync"

### DISCUSSION THREAD

**Discussion by neingram** [windingwind/zotero-better-notes#838] [Q&A] [answered]
Hello! When I try to export my note to set-up auto sync, it wouldn't allow me to click on the box to do so (see [this screencast)](https://www.loom.com/share/0d44988381184ea995d2971e41fd6f2b). Any help is much appreciated!

**Comment by windingwind** [comment, marked as answer]
Added a tooltip in the latest beta:
<img width="400" alt="image" src="https://github.com/windingwind/zotero-better-notes/assets/33902321/7e5c7593-e575-4b95-bc3d-7838e2fe08c4">

**Reply by neingram** [reply]
Thank you so much!
